pallets/capacity/src/lib.rs

```diff
@@ -126,29 +142,138 @@ pub mod pallet {
 
 	#[pallet::event]
 	#[pallet::generate_deposit(pub (super) fn deposit_event)]
-	pub enum Event<T: Config> {}
+	pub enum Event<T: Config> {
+		/// Tokens have been staked to the Frequency network.
+		Staked {
+			/// The token account that staked tokens to the network.
+			account: T::AccountId,
+			/// The MSA that a token account targeted to receive Capacity based on this staking amount.
+			target: MessageSourceId,
+			/// An amount that was staked.
+			amount: BalanceOf<T>,
```

Comment:
Do we need to add a `capacity` field here to match the design doc?

Reply:
Ahh, you are right. Can I do it in a different PR so that you are not longer blocked? I also wanted to run some benchmarks that might allow for a simpler way to accomplish based on learning that I get from it.

Reply:
https://github.com/LibertyDSNP/frequency/issues/891